Tests/LibWeb/Layout/expected/table/avoid-div-by-zero-in-table-measures.txt

```diff
@@ -27,7 +27,7 @@ ViewportPaintable (Viewport<#document>) [0,0 800x600]
     PaintableWithLines (BlockContainer<BODY>) [8,8 784x23]
       PaintableWithLines (TableWrapper(anonymous)) [8,8 6x23] overflow: [8,8 17.265625x23]
         PaintableBox (Box<TABLE>) [8,8 6x23] overflow: [8,8 17.265625x23]
-          PaintableBox (Box<TBODY>) [8,8 0x19] overflow: [10,10 15.265625x20]
-            PaintableBox (Box<TR>) [10,10 0x19] overflow: [10,10 15.265625x20]
+          PaintableBox (Box<TBODY>) [10,10 2x19] overflow: [10,10 15.265625x20]
+            PaintableBox (Box<TR>) [10,10 2x19] overflow: [10,10 15.265625x20]
```

Comment:
Also here

Reply:
I opened up a bug today regarding this https://github.com/LadybirdBrowser/ladybird/issues/1790